src/main/java/com/princexml/wrapper/Prince.java

```diff
@@ -15,6 +15,9 @@
 
 import static com.princexml.wrapper.CommandLine.*;
 
+/**
+ * The main Prince class.
```

Comment:
Perhaps "main Prince class" doesn't quite clarify how it differs from PrinceControl?

Reply:
I'll provide a better description. Also, should this class be renamed to `PrinceCommandLine`?

Reply:
Not sure, that feels a bit unwieldy and `Prince` is pleasingly concise, so I think we can stick with it for now.